lib/membrane_h264_ffmpeg/common.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Membrane.H264.FFmpeg.Common do
+  @h264_time_base 90_000
```

Comment:
If you want to hide the module from docs, add `@moduledoc false` and convert the comments before function into regular `@doc`